sips/aggregator_committee_consensus.md

```diff
@@ -223,6 +229,67 @@ type PartialSignatureMessage struct {
+    ValidatorRegistrationPartialSig // 4
+    // VoluntaryExitPartialSig is a partial signature over a VoluntaryExit object
+    VoluntaryExitPartialSig // 5
+    // AggregatorCommitteePartialSig is a partial signature for combined aggregator and sync committee selection proofs
+    AggregatorCommitteePartialSig // 6
+)
+```
+
+
+#### Pre-Consensus Phase
+
+Differently from the `CommitteeRunner` used for attestations and sync-committee duties,
+the `AggregatorCommitteeRunner` includes a pre-consensus phase.
+In such a phase, operators exchange a `PartialSignatureMessages`
+with one `PartialSignatureMessage` for each validator duty in the committee.
+Note that, while a possible aggregator duty incurs only one `PartialSignatureMessage`,
+a validator may be assigned to up to four sync committee contribution duties,
+thus possibly requiring four `PartialSignatureMessages`.
+The `PartialSignatureMessages.Type` is set to the new type `AggregatorCommitteePartialSig`.
+
+The pre-consensus phase ends on the following scenarios:
+1. All validators are confirmed not to be selected as an aggregator/contributor.
+2. At least one validator is confirmed to be selected as an aggregator/contributor.
+
+> [!TIP] 
+> Note that, according to the weak condition (2), even if the selection cannot
+be determined for a subset of validators, the pre-consensus phase
+will still proceed to the consensus phase and these validators won't be included on it.
+> Although it seems counter-intuitive to do so,
+> that's necessary for ensuring liveness,
+> in a context with BFT assumptions
+> and due to the possibility of invalid signatures/validators miss
+> associated with different smart-contract state views.
+> 
+> How can this happen in more concrete terms?
+> Let $V_i$ be the validators set (of the committee) in the perspective of operator $O_i$.
+> - Operators have different views of the current SSV smart-contract state (e.g. only one is aware of a new validator that joined the committee).
+> In this case, the operator $O_i$ will only possibly select validators $V_i \cap (\bigcup_{j\neq i}V_j)$.
+> - Suppose byzantine operators send a valid signature only for $v\in V_e$ and invalid ones for all other validators.
+> Once a quorum is reached, honest operators may be restricted to advance to consensus only with the duty for $v$.
+> Note that in case it receives an all-honest quorum, it may still be able to advance to consensus with more validators than just $v$.
+> 
+> While this solution prioritizes liveness over completeness,
+> it's acceptable for now as the worst-case scenarios are marginally expected.
+> Still, our context may be formally linked to the *Interactive Consistency*
+> problem, presented in the [*Reaching Agreement in the Presence of Faults*](https://lamport.azurewebsites.net/pubs/reaching.pdf)
+> paper, and more robust solutions may be explored in the future ([example](https://cgi.di.uoa.gr/~mema/publications/ic-extended.pdf)).
```

Comment:
good!
you can farther downplay this problem by explaining that it is hard to exploit, since the attacker must know ahead of time what validators it holds can be selected